pkg/vz/errors_darwin.go

```diff
@@ -7,5 +7,4 @@ package vz
 
 import "errors"
 
-//nolint:revive // error-strings
-var errRosettaUnsupported = errors.New("Rosetta is unsupported on non-ARM64 hosts")
+var errRosettaUnsupported = errors.New("rosetta is unsupported on non-ARM64 hosts")
```

Comment:
This change is incorrect. "Rosetta" is a proper noun and therefore [should be capitalized](https://go.dev/wiki/CodeReviewComments#error-strings):

> Error strings should not be capitalized (unless beginning with proper nouns or acronyms)

Reply:
Thanks.